src/main/java/uk/gov/pay/api/exception/mapper/SearchChargesExceptionMapper.java

```diff
@@ -9,19 +9,29 @@
 import javax.ws.rs.ext.ExceptionMapper;
 
 import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+import static javax.ws.rs.core.Response.Status.NOT_FOUND;
 import static uk.gov.pay.api.model.PaymentError.Code.SEARCH_PAYMENTS_CONNECTOR_ERROR;
+import static uk.gov.pay.api.model.PaymentError.Code.SEARCH_PAYMENTS_NOT_FOUND;
 import static uk.gov.pay.api.model.PaymentError.aPaymentError;
 
 public class SearchChargesExceptionMapper implements ExceptionMapper<SearchChargesException> {
-
     private static final Logger LOGGER = LoggerFactory.getLogger(SearchChargesExceptionMapper.class);
 
     @Override
     public Response toResponse(SearchChargesException exception) {
-        PaymentError paymentError = aPaymentError(SEARCH_PAYMENTS_CONNECTOR_ERROR);
-        LOGGER.error("Connector invalid response was {}.\n Returning http status {} with error body {}", exception.getMessage(), INTERNAL_SERVER_ERROR, paymentError);
+        PaymentError paymentError = null;
```

Comment:
Fine to use, but I don't find it of much use, i.e. new template, people are just filling what and other info is of no use. Above is what explains what the PR is about